.github/workflows/cd-app-deployment.yml

```
@@ -26,7 +26,7 @@ jobs:
     with:
       app_name: ${{ github.event.inputs.app_name }}
       image_tag: ${{ github.event.inputs.image_tag }}
-      environment_name: dev
+      environment_name: pr50
```

Reminder to reset this back to `dev`
```suggestion
      environment_name: dev
```